Failed to build with latest Gala

### What Happened?

Wingpanel fails to build with Gala after https://github.com/elementary/gala/commit/b89ec3a1fd1e3e1f2b585ba8af25d3be950e3c45.


### Steps to Reproduce

1. Install Gala after https://github.com/elementary/gala/commit/b89ec3a1fd1e3e1f2b585ba8af25d3be950e3c45 (or use latest OS 8 Daily)
2. Clone and build latest Wingpanel 31608767dc6644e1e4e698680038b2f4cf473ece

### Expected Behavior

The build succeeds.


### OS Version

8.x (Early Access)

### OS Architecture

arm64 (on Raspberry Pi, Apple Silicon, etc.)

### Session Type

Secure Session (Wayland)

### Software Version

Latest release (I have run all updates)

### Log Output

```shell
ryo@elementary-daily:~/work/wingpanel (master =)$ ninja -C builddir/
ninja: Entering directory `builddir/'
[1/7] Compiling Vala source ../wingpanel-interface/Main.vala ../wingp...ngpanel-interface/FocusManager.vala ../wingpanel-interface/Utils.vala
FAILED: wingpanel-interface/libwingpanel-interface.so.p/Main.c wingpanel-interface/libwingpanel-interface.so.p/DBusServer.c wingpanel-interface/libwingpanel-interface.so.p/BackgroundManager.c wingpanel-interface/libwingpanel-interface.so.p/FocusManager.c wingpanel-interface/libwingpanel-interface.so.p/Utils.c wingpanel-interface/wingpanel-interface.h wingpanel-interface/wingpanel-interface.vapi
valac -C --debug --debug --vapidir=/home/ryo/work/wingpanel/vapi --pkg mutter-clutter-14 --pkg mutter-cogl-pango-14 --pkg mutter-cogl-14 --pkg mutter-mtk-14 --pkg libmutter-14 --pkg granite --pkg gala --color=always --directory wingpanel-interface/libwingpanel-interface.so.p --basedir ../wingpanel-interface --library wingpanel-interface --header wingpanel-interface/wingpanel-interface.h --vapi ../wingpanel-interface.vapi --define HAS_MUTTER46 --define VALA_0_56_17 ../wingpanel-interface/Main.vala ../wingpanel-interface/DBusServer.vala ../wingpanel-interface/BackgroundManager.vala ../wingpanel-interface/FocusManager.vala ../wingpanel-interface/Utils.vala
../wingpanel-interface/FocusManager.vala:120.25-120.45: error: 1 missing arguments for `Gala.ModalProxy Gala.WindowManager.push_modal (Clutter.Actor, bool)'
  120 |             var proxy = wm.push_modal (stage);
      |                         ^~~~~~~~~~~~~~~~~~~~~
Compilation failed: 1 error(s), 0 warning(s)
ninja: build stopped: subcommand failed.
[ble: exit 1]
ryo@elementary-daily:~/work/wingpanel (master =)$
```

### Hardware Info

_No response_

--- a/README.md
+++ b/README.md
@@ -7,7 +7,7 @@ The extensible top panel for Pantheon. Wingpanel is an empty container that acce
 
 You'll need the following dependencies:
 
-* libgala-dev
+* libgala-dev >= 8.3.0
 * libgee-0.8-dev
 * libglib2.0-dev
 * libgranite-dev >= 5.4.0

--- a/wingpanel-interface/meson.build
+++ b/wingpanel-interface/meson.build
@@ -59,7 +59,7 @@ endif
 
 mutter_typelib_dir = libmutter_dep.get_variable('typelibdir')
 
-gala_dep = dependency('gala')
+gala_dep = dependency('gala', version: '>= 8.3.0')
 m_dep = meson.get_compiler('c').find_library('m', required: false)
 
 shared_module(

